metadata-ingestion/src/datahub/ingestion/source/dbt.py

```diff
@@ -473,6 +474,7 @@ def get_upstream_lineage(upstream_urns: List[str]) -> UpstreamLineage:
     "timestamp without time zone": DateTypeClass,
     "integer": NumberTypeClass,
     "float8": NumberTypeClass,
+    "struct": MapTypeClass,
```

Comment:
I think this should be RecordType!

Reply:
In glue source struct is considered `MapTypeClass`. Is that also wrong? I took example from there.